astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -180,6 +180,10 @@ def draw(self, renderer, bboxes, ticklabels_bbox):
 
                     dist = np.hypot(dx, dy)
 
+                    # Avoid RuntimeWarning
+                    if dist == 0 or not np.isfinite(dist):
```

Comment:
Just let me know when you are done waiting for reply and want me to take this out.

Also, should we increase `remote_timeout` here or in a separate PR? Thinking about it more, that one is more like an API change and probably should have a change log.